web/src/server/api/routers/generations/getAllQuery.ts

```diff
@@ -61,7 +41,7 @@ export const getAllQuery = protectedProjectProcedure
                 scores."project_id" = ${input.projectId}
                 AND scores."trace_id" = t.id
                 AND scores."observation_id" = o.id
```

Comment:
dropping categorical, check also in other files that you only drop these for filters

Reply:
Filter only, all good 👍